packages/passkeys/passkeys/example/README.md

```diff
@@ -1,3 +1,10 @@
 # passkeys_example
 
-Demonstrates how to use the passkeys plugin.
+This example demonstrates how to use the passkeys plugin.
```

Comment:
I think we need to conistently speak of "passkeys plugin" or "passkeys package" - so far, I prefer and used the latter.